README.md

```diff
@@ -13,6 +13,8 @@ Get the source using one of the following:
 - Download [`build/clipboard-polyfill.js`](https://raw.githubusercontent.com/lgarron/clipboard-polyfill/master/build/clipboard-polyfill.js) and include it using a `<script>` tag.
 - `npm install clipboard-polyfill` and import as `clipboard`.
 
+Please note that this library uses ES6 features, notably [`Promise`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise), so if you need to support environments that do not natively support `Promise` (ie. IE <= 11) you will need to include that dependency separately. Recommendations include [es6-promise](https://github.com/stefanpenner/es6-promise) and [core-js](https://github.com/zloirock/core-js).
```

Comment:
I specifically avoid using ES6 features and anything that doesn't work in IE. Promises are the only exception, because the API format relies on it. Could you rephrase to reflect that?